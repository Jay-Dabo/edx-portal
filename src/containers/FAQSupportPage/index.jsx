import React from 'react';
import { Helmet } from 'react-helmet';
import { MailtoLink } from '@edx/paragon';

import H1 from '../../components/H1';

const FAQSupportPage = () => (
  <div className="container">
    <Helmet>
      <title>Support</title>
    </Helmet>
    <H1>Support</H1>
    <p>
      For assistance, please contact edX Enterprise Support at <MailtoLink to="enterprise@edx.org" content=" enterprise@edx.org" />.
    </p>
  </div>
);

export default FAQSupportPage;
