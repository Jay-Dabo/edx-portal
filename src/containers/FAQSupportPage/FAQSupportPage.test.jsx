import React from 'react';
import renderer from 'react-test-renderer';

import FAQSupportPage from './index';

describe('<FAQSupportPage />', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create((
        <FAQSupportPage />
      ))
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
