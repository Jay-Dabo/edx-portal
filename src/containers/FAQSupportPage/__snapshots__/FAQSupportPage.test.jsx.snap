// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FAQSupportPage /> renders correctly 1`] = `
<div
  className="container"
>
  <h1>
    FAQ & Support
  </h1>
  <p>
    Blah blah blah blah blah blah
  </p>
</div>
`;
